lib/validators/options_processor/teams.js

```diff
@@ -47,4 +67,35 @@ const getTeamMembers = async (context, team) => {
   return res.data.map(member => member.login)
 }
 
+const getTeamMembershipState = async (context, team, username) => {
+  const stringArray = team.split('/')
+  if (stringArray.length !== 2) {
+    throw Error(`each team id needs to be given in 'org/team_slug'`)
+  }
+  if (stringArray[0].indexOf('@') === 0) stringArray[0] = stringArray[0].substring(1)
```

Comment:
Looks like you are supporting both naming prefix with `@` and without here. i.e.`@org/teamName` and `org/teamName` here. 

Good call.

To remove the first element, you may want to consider using `stringArray.shift()` here.

Reply:
In fact, I haven't add this but just copy/paste from old code. Do you want me to replace again ?

Reply:
I do think it's more readable with `shift`. If you agree, please do. Not a big deal otherwise.